carmm/run/workflows/react.py

```diff
@@ -548,6 +609,120 @@ def vibrate(self, atoms: Atoms, indices: list, read_only=False):
             vib.write_mode()
             return vib
 
+    @property
+    def MaceReact_Preoptimiser(self):
+        """
+        Get or set the MACE preoptimiser used in aims_optimise or ts_search with setting
+
+        Args:
+             MaceReact MaceReact Obj:
+               User-defined MaceReact.
+
+        Returns:
+             self._MaceReactor MaceReact Obj:
+               User-defined MaceReact.
+        """
+
+        return self._MaceReactor
+
+    @MaceReact_Preoptimiser.setter
+    def MaceReact_Preoptimiser(self, MaceReact):
+
+        self._MaceReactor = MaceReact
+
+    def _mace_preoptimise(self, atoms: Atoms, fmax, relax_unit_cell, optimiser, opt_kwargs = {}):
+        """
+        Internal function for preoptimisation of structures with a pre-set MACE calculator
+
+        Args:
+            atoms: Atoms object
+            fmax: flaot
```

Comment:
Typo.

Reply:
Consider making it clear what these are for the next user - or point to where clearly defined.

Reply:
Cheers! Both resolved.